code/game/objects/items/weapons/melee/energy_melee_weapons.dm

```diff
@@ -350,7 +360,7 @@
 			icon_state = "sword[item_color]"
 			set_light(brightness_on, l_color=colormap[item_color])
 		w_class = w_class_on
-		playsound(user, 'sound/magic/fellowship_armory.ogg', 35, TRUE, frequency = 90000 - (active * 30000))
+		playsound(user, 'sound/magic/fellowship_armory.ogg', 35, TRUE, frequency = 60000)
```

Comment:
can you restore this? the different clanging sounds are pretty iconic for this item.